NuRadioReco/detector/RNO_G/rnog_detector.py

```diff
@@ -1302,9 +1302,15 @@ def get_time_delay(self, station_id, channel_id, use_stored=True, trigger=False)
         time_delay: float
             Sum of the time delays of all components in the signal chain for one channel
 
+        Important note
```

Comment:
According to our documentation we should only use sections which are defined in the [numpydoc docs](https://numpydoc.readthedocs.io/en/latest/format.html). So in this case you could use `Notes`. 
Maybe do:
```
Notes
------
IMPORTANT: bla bla
```